patches/ui-webui-resources-cr_elements-cr_toolbar-cr_toolbar.html.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.html b/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.html
+index 8fb18010132c64a0ed6ff19357e43642ad9c2311..fcf7ae4023558853f62a8231c17db8620741726b 100644
+--- a/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.html
++++ b/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.html
+@@ -133,7 +133,7 @@
+     <div id="leftContent">
+       <div id="leftSpacer">
+         <!-- Note: showing #menuPromo relies on this dom-if being [restamp]. -->
+-        <template is="dom-if" if="[[showMenu]]" restamp>
++        <!-- <template is="dom-if" if="[[showMenu]]" restamp>
```

Comment:
if this is based on `showMenu` then why don't we just turn off `showMenu`?

Reply:
showMenu is a function on the prototype for MenuButton. So I think we'll have to extend and set to undefined.